pkg/gofr/migration/migration.go

```diff
@@ -23,6 +23,8 @@ func Run(migrationsMap map[int64]Migrate, c *container.Container) {
 		return
 	}
 
+	c.Debug("sorting the migration keys")
```

Comment:
Is the user aware that migrations will be executed in sorted order of keys?

Reply:
partly yes, but no. We asks user to create migrations file in timestamp format (so that way, one can say that user is aware about sorting). But not directly aware of how gofr is handling the keys.

Reply:
So, this log might help them to know that the keys are getting sorted and migrations will run in sorted order?